### ISSUE TITLE
IPT3 functional testing - parts of mandatory metadata not filled out when uploading a resource as metadata, but is registered if metadata is uploaded as part on an archive

### ISSUE THREAD

**Issue by CecSve** [gbif/ipt#2307]
This issue is based on the MICA test dataset. 

If you upload an archive, the metadata appears to be filled out for the mandatory contributor fields `title`, `first name` and `last name`. However, if the resource is first created based on a metadata file only, and source files are added afterwards, then the mandatory fields `title`, `first name` and `last name` are not filled out in the contributor's metadata. 

It seems that the structure of the two metadata files is not identical, and it does not work to upload the metadata of the archive as a metadata-only resource. What are the differences between the two files and where is it documented? And can the IPT be updated to read the contributor information automatically if the resource is constructed as metadata initially?

**Comment by mike-podolskiy90** [comment]
No, it's supposed to work the same. I'll see what is wrong there

**Comment by ManonGros** [comment]
I encountered the same issue.
